react/ContactsList/ContactRow.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import withBreakpoints from '../helpers/withBreakpoints'
+import cx from 'classnames'
+
+import contactPropTypes from './ContactPropTypes'
+
+import ContactPhone from './Contacts/ContactPhone'
+import ContactIdentity from './Contacts/ContactIdentity'
+import ContactCozy from './Contacts/ContactCozy'
+import ContactEmail from './Contacts/ContactEmail'
+import styles from './styles.styl'
+
+const getPrimaryOrFirst = (arr = [{}]) => arr.find(obj => obj.primary) || arr[0]
```

Comment:
Come on, we never used back our Contact model? :'( 

https://github.com/cozy/cozy-libs/blob/master/packages/cozy-doctypes/src/contacts/Contact.js#L7